gravitybridge/assetlist.json

```diff
@@ -295,6 +295,41 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/dai.svg"
         }
       ]
+    },
+    {
+      "description": "Gravity Bridge OCC",
+      "denom_units": [
+        {
+          "denom": "gravity0x2F109021aFe75B949429fe30523Ee7C0D5B27207",
+          "exponent": 0
+        },
+        {
+          "denom": "gocc",
+          "exponent": 18
+        }
+      ],
+      "base": "gravity0x2F109021aFe75B949429fe30523Ee7C0D5B27207",
+      "name": "OccamFi",
+      "display": "gocc",
+      "symbol": "OCC",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/gravitybridge/images/occamfi.png"
+      },
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "0x2f109021afe75b949429fe30523ee7c0d5b27207"
```

Comment:
The capitalization needs to match. I can see '0x2F...' as the base in ethereum but '0x2f...' here

Reply:
fixed. thank you